recipes/rapidjson/all/conanfile.py

```diff
@@ -20,10 +20,16 @@ def layout(self):
         basic_layout(self, src_folder='src')
         self.folders.build = 'build'
         self.folders.generators = 'build/conan'
+        if self.settings.os == 'Windows' and self.settings.arch == 'armv8':
+            os.environ["CXXFLAGS"] = '-DRAPIDJSON_ENDIAN=RAPIDJSON_LITTLEENDIAN'
```

Comment:
I wonder if this line is needed, but I don't see any harm in it either.

I think that the patch to `rapidjson.h` is the reason it works.